cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -1322,7 +1322,7 @@ export function trackAnnotationsAsync(sessionInstance: any, frame: number):
         const { tracked_shapes: trackedShapes } = await sessionInstance.annotations.computeTrackingData({
             job_id: sessionInstance.id,
             shape_tracks: trackedPoints,
-            stop_frame: frame + 10,
+            stop_frame: frame + 5,
```

Comment:
reason?

Reply:
reasoning could even be in commit message (for the future), but to the outside worlds this value is entirely unclear

Reply:
completely :sweat_smile: , this was a request from ahmed. i think we will parametrise this in the future.

this is the number of frames that will be tracked in the backend, the more the better for the annotator, but slows the interface, because the request takes longer.